etcdctl/ctlv3/command/global.go

```diff
@@ -79,6 +83,14 @@ func initDisplayFromCmd(cmd *cobra.Command) {
 func mustClientFromCmd(cmd *cobra.Command) *clientv3.Client {
 	flags.SetPflagsFromEnv("ETCDCTL", cmd.InheritedFlags())
 
+	debug, derr := cmd.Flags().GetBool("debug")
+	if derr != nil {
+		ExitWithError(ExitError, derr)
```

Comment:
It seems to me that `ExitError` is 0. Why are we exiting the program with the exit value as 0 but also write to stderr? I am just curious.